hw/ip/aes/model/crypto.h

```diff
@@ -14,7 +14,8 @@ typedef enum crypto_mode {
   kCryptoAesCfb = 1 << 2,
   kCryptoAesOfb = 1 << 3,
   kCryptoAesCtr = 1 << 4,
-  kCryptoAesNone = 1 << 5
+  kCryptoAesGcm = 1 << 5,
+  kCryptoAesNone = 1 << 6
```

Comment:
We've changed this to not alter the software interface of AES and maintain binary compatibility. `kCryptoAesNone` is now 6'b11_1111 instead of 7'b100_0000 (initial approach) or 6'b10_0000 (original design, now used for GCM).